python/cudf/udf_cpp/CMakeLists.txt

```diff
@@ -56,7 +72,7 @@ target_link_libraries(cudf_strings_udf PUBLIC cudf::cudf)
 install(TARGETS cudf_strings_udf DESTINATION ./cudf/_lib/)
 
 # Create the shim library for each architecture.
-set(SHIM_CUDA_FLAGS --expt-relaxed-constexpr -rdc=true)
+set(SHIM_CUDA_FLAGS --expt-relaxed-constexpr -rdc=true --expt-extended-lambda)
```

Comment:
Do you know exactly what requires the extended lambdas here? I think we are trying to move away from using that flag everywhere in libcudf.

Reply:
I couldn't remember any reason for this so I removed it and nothing broke. Probably was hanging around from some debugging issue earlier in development.